### PR TITLE
Fix shell Services e2e test

### LINKED ISSUE #10127: Flaky e2e: Shell should pass tests for services.sh (Failed 3 times in the last 30 runs. Stability: 90 %)
Master issue is #9312 

Recent failures include:

====

waiting for service3 at 10.0.193.5:80
wget: download timed out
ERROR: (gcloud.compute.ssh) [/usr/bin/ssh] exited with return code [1].
testing service3 VIP from container failed
namespaces/services-test-701

===

waiting for service3 at 10.0.110.153:80
Test 7: Bring up another service, make sure it does not re-use IPs.
Starting service 'services-test-15187/service4' on port 80 with 3 replicas
error: could not read an encoded object from STDIN: unable to connect to a server to handle "replicationcontrollers": couldn't read version from server: Get https://173.255.114.153/version: dial tcp 173.255.114.153:443: connection refused

error: no objects passed to create
error: couldn't read version from server: Get https://173.255.114.153/version: dial tcp 173.255.114.153:443: connection refused

====

### PR DESCRIPTION
**Pull request by wojtek-t**
Fixes #10127

For more details see: https://github.com/GoogleCloudPlatform/kubernetes/issues/10127#issuecomment-120857990

cc @davidopp

### CODE REVIEW

**Comment by wojtek-t** [comment]
cc @gmarek

**Comment by gmarek** [comment]
LGTM

**Comment by wojtek-t** [comment]
cc @zmerlynn

File: hack/e2e-suite/services.sh [hunk 1]
```diff
@@ -79,6 +79,22 @@ function make_namespace() {
 __EOF__
 }
 
+wait_for_apiserver() {
+  echo "Waiting for apiserver to be up"
+
+  local i
+  for i in $(seq 1 12); do
+    results=($(ssh-to-node "${master}" "
+      wget -q -T 1 -O - http://localhost:8080/healthz || true
```
Comment (zmerlynn) [inline hack/e2e-suite/services.sh]: This test is already using `curl`.. I know it's slightly pedantic, but can we stick to one?

**Comment by zmerlynn** [comment]
Total nit, but LGTM otherwise.

File: hack/e2e-suite/services.sh (hunk 1 above)
Comment (wojtek-t) [inline hack/e2e-suite/services.sh]: It is using wget in more places (e.g. in busybox there is not curl).
BTW - I'm already working on rewriting this test in GO - see #11056 (hopefully will finish it tomorrow) so I would prefer not spend too much time on this one.

Comment (zmerlynn) [inline hack/e2e-suite/services.sh]: Ah, you're right, it's already mixed even on the ssh case.

(I was ignoring the busybox container case - there you definitely have to be careful.)

**Comment by zmerlynn** [comment]
LGTM

**Pull request merged**